acoular/grids.py

```diff
@@ -823,98 +981,129 @@ def _get_subgrids(self):
+
+        This method determines if each column of the input array :attr:`pos` corresponds to a point
+        that falls within the sector. For this base class, all points are considered within the
+        sector.
 
         Parameters
         ----------
-        pos : array of floats
-            Array with the shape 3x[number of gridpoints] containing the
-            grid positions
+        pos : numpy.ndarray
+            A 2D array with shape (3, N), where N is the number of grid points. Each column
+            represents the x, y, z coordinates of a grid point.
 
         Returns
         -------
-        array of bools with as many entries as columns in pos
-            Array indicating which of the given positions lie within the
-            given sector
-
+        numpy.ndarray
+            A 1D boolean array of length N, where each entry indicates whether the corresponding
+            column in :attr:`pos` lies within the sector's bounds.
+
+        Examples
+        --------
+        >>> import numpy as np
+        >>> from acoular import grids
+        >>> sector = grids.Sector()
+        >>> positions = np.array([[0, 1], [0, 0], [0, 0]])  # Two grid points
+        >>> sector.contains(positions)
+        array([ True,  True])
         """
         return ones(pos.shape[1], dtype=bool)
 
 
 class SingleSector(Sector):
-    """Base class for single sector types.
+    """
+    Base class for single sector types.
 
-    Defines the common interface for all single sector classes. This class
-    may be used as a base for diverse single sector implementations. If used
-    directly, it implements a sector encompassing the whole grid.
+    Defines the common interface for all single sector classes. This class can serve as a base for
+    various single sector implementations. When used directly, it defines a sector that encompasses
+    the whole grid. It includes attributes for handling border inclusion, tolerance for sector
+    borders, and default behavior when no points are inside the sector.
     """
 
-    #: Boolean flag, if 'True' (default), grid points lying on the sector border are included.
+    #: If `True`, grid points lying on the sector border are included in the sector. Default is
+    #: `True`. (optional)
     include_border = Bool(True, desc='include points on the border')
 
-    #: Absolute tolerance for sector border
+    #: The absolute tolerance to apply when determining if a grid point lies on the sector border.
+    #: Default is `1e-12`. (optional)
     abs_tol = Float(1e-12, desc='absolute tolerance for sector border')
 
-    #: Boolean flag, if 'True' (default), the nearest grid point is returned if None is inside the
-    #: sector.
+    #: If `True`, the nearest grid point is returned if no grid points are inside the sector.
+    #: Default is `True`. (optional)
     default_nearest = Bool(True, desc='return nearest grid point to center of none inside sector')
```

Comment:
There is a typo, but even when fixing this, the sphinx docstring and trait description are unclear. `default_nearest` is an attribute, not a function. Thus, it can not return anything. Instead, it should mention its purpose and the method (`contains`) where the attribute becomes relevant. 

```suggestion
    default_nearest = Bool(True, desc='return nearest grid point to center if none inside sector')
```